kin-ecosystem-core/src/main/java/kin/ecosystem/core/util/StringUtil.java

```diff
@@ -41,4 +42,9 @@ public static String toIndentedString(Object o) {
   public static String getAmountFormatted(int amount) {
     return NumberFormat.getNumberInstance(Locale.US).format(amount);
   }
+
+  public static String getAmountFormatted(String amount) {
```

Comment:
this is dangerous? what if the string is is not a number?

Reply:
the method implies for a number.